### PR TITLE
KOGITO-4346: Move VSCode UI test execution into it-tests github workflow

### PR DESCRIPTION
**Pull request by domhanak**
Add new job for vscode it tests
Move the tests execution from build:prod
Rename script name from ui-test to test:it
Add archiving of artifacts for it tests of vscode

cc @tiagobento @jstastny-cz 
Videos and screenshot are not possible as of now :)

### CODE REVIEW

**Review by jstastny-cz — APPROVED** [review]

**Comment by domhanak** [comment]
@paulovmr FDB fails on kie-wb-common build, unrelated. Ready for merging when approved from your side :)

**Comment by manstis** [comment]
It doesn't look like https://github.com/kiegroup/kogito-tooling/pull/409 will be affected.

However please confirm if I need to make any changes for the new IT tests in the referenced PR.

**Comment by domhanak** [comment]
@manstis Yes #409 should be good, only thing that will change is the command to run the IT tests `yarn run test:it`

File: .github/workflows/it-tests.yml [hunk 1]
```diff
@@ -123,3 +123,46 @@ jobs:
         with:
           name: kie-editors-standalone-it-tests-videos-${{ matrix.os }}
           path: packages/kie-editors-standalone/it-tests/cypress/videos
+
+  kie-editors-vscode-tests-it:
```
Comment (tiagobento) [inline .github/workflows/it-tests.yml]: Can you please rename it to `vscode-extension-tests-it`?

**Review by tiagobento — APPROVED** [review]

**Pull request merged**